handler/population-types.go

```diff
@@ -32,3 +35,37 @@ func (h *PopulationTypes) Get(w http.ResponseWriter, req *http.Request) {
 		h.responder.JSON(ctx, w, http.StatusOK, body)
 	}
 }
+
+// Get is the handler for GET /area-types
+func (h *PopulationTypes) GetAreaTypes(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	req := cantabular.GetGeographyDimensionsRequest{
+		Dataset: chi.URLParam(r, "population-type"),
+	}
+
+	res, err := h.cantabularClient.GetGeographyDimensions(ctx, req)
+	if err != nil {
+		h.responder.Error(
+			ctx,
+			w,
+			dperrors.StatusCode(err), // Can be changed to ctblr.StatusCode(err) once added to Client
```

Comment:
This comment can be deleted and you can use cantabularClient.StatusCode(err) now

Reply:
This has been addressed